src/inventory.py

```diff
@@ -1,14 +1,19 @@
-"""The Inventory singleton"""
+"""
+src/inventory.py
+================
+
+The Inventory singleton
+.. todo:: make imports dynamic, and watch out for frozen, like with messagetypes
```

Comment:
Incorrect. This TODO was originally for `import storage.sqlite`.

Reply:
Well it refers to imports of children, not only `storage.sqlite`.